client.py

```diff
@@ -0,0 +1,197 @@
+""" Shest Gui Client """
+
+
+from CONSTANTS import DOWNLOAD_PATH, UPLOAD_PATH, REGULAR_PATH, \
+    DOWNLOAD_COMMAND, UPLOAD_COMMAND, KEY
+from cryptography.fernet import Fernet
+from pathlib import Path
+from re import search
+import requests
+import time
+import tkinter
+from typing import List
+
+
+def window_configuration() -> None:
```

Comment:
`window_configuration` sounds more like a variable name than a function name. Rename it to `configure_window`.